test/liquid/strict_parse_test.exs

```diff
@@ -42,15 +42,19 @@ defmodule Liquid.StrictParseTest do
   end
 
   test "missing endtag parse time error" do
-    assert_raise RuntimeError, "Invalid tag name", fn ->
-      Template.parse("{% for a in b %} ...")
-    end
+    assert_raise RuntimeError,
+                 "Invalid tag name for, 1.-if the tag is a block is malformed  2.- the same name of a liquid tag for a custom tag",
```

Comment:
change error message to: "Invalid tag name. The tag block is malformed or you are using a reserved tag name to define a Custom Tag".

Reply:
done